src/posthog-surveys.ts

```diff
@@ -74,6 +71,7 @@ export class PostHogSurveys {
             if (this._surveyEventReceiver == null) {
                 this._surveyEventReceiver = new SurveyEventReceiver(this.instance.persistence)
             }
```

Comment:
there's some kind of setup/teardown confusion happening here - probably/hopefully just me being slow